src/strgen/strgen_base.cpp

```diff
@@ -419,6 +422,24 @@ void EmitGender(Buffer *buffer, char *buf, int)
 	}
 }
 
+void EmitCargoList(Buffer *buffer, char *buf, int)
+{
+	/* This is a {CARGO_LIST " and " ", "} command. */
+	const char *last_separator = ", "; // or " and " / ", and " / " or " / ", or "
+	const char *separator = ", ";
+
+	const char *word = ParseWord(&buf);
+	if (word != nullptr) {
+		last_separator = word;
+		word = ParseWord(&buf);
+		if (word != nullptr) separator = word;
+	}
+
+	buffer->AppendUtf8(SCC_CARGO_LIST);
+	EmitWord(buffer, separator);
+	EmitWord(buffer, last_separator);
+}
```

Comment:
As an English-pedant, I would like Oxford-comma support:

1; Passengers
2: Passengers and Mail
3: Passengers, Mail, and Goods

There may be other rules in other languages too though.

Reply:
Though shouldn't the newspaper headlines/texts follow the style guides of British newspapers?

The Times: "Avoid the so-called Oxford comma" (https://web.archive.org/web/20080724151906/http://www.timesonline.co.uk/tol/tools_and_services/specials/style_guide/article986734.ece)
The Guardian: "a comma before the final “and” in lists: straightforward ones [...] do not need one" (https://www.theguardian.com/guardian-observer-style-guide-o)
Fun thing: some newspapers/headlines occasionally omit the "and" completely in favour of a comma to shorten texts.

Not to mention that "The Oxford Style Manual" requires the Oxford-comma, whereas the "University of Oxford Public Affairs Directorate Writing and Style Guide" generally dislikes the Oxford-comma unless it reduces ambiguity.